packages/clerk-js/src/ui/components/devPrompts/EnableOrganizationsPrompt/index.tsx

```diff
@@ -0,0 +1,247 @@
+import type { __internal_EnableOrganizationsPromptProps } from '@clerk/shared/types';
+import { css } from '@emotion/react';
+
+import { Modal } from '@/ui/elements/Modal';
+import { InternalThemeProvider } from '@/ui/styledSystem';
+
+import { Flex } from '../../../customizables';
+import { Portal } from '../../../elements/Portal';
+import { basePromptElementStyles, PromptContainer } from '../shared';
+
+const EnableOrganizationsPromptInternal = (props: __internal_EnableOrganizationsPromptProps) => {
+  const ctaText = 'componentName' in props ? `<${props.componentName} /> component` : props.utilityName;
+
+  return (
+    <Portal>
+      <Modal
+        canCloseModal={false}
+        contentSx={() => ({ animation: 'unset' })}
+      >
+        <PromptContainer
+          sx={() => ({
+            top: '50%',
+            left: '50%',
+            bottom: 'auto',
+            right: 'auto',
+            padding: 'unset',
+            transform: 'translate(-50%, -50%)',
+            flexDirection: 'column',
+            height: 'fit-content',
+          })}
```

Comment:
We can remove all of this, if PromptContainer becomes more generic with its styles.